libraries/engage/components/Toggle/index.jsx

```diff
@@ -15,9 +15,17 @@ const Toggle = ({
 }) => (
   <div className={className}>
     <div className={container}>
-      <input type="checkbox" id={id} className={classNames(checkbox, hidden)} checked={checked} onChange={onChange} disabled={disabled} />
-      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-      <label htmlFor={id} className={label} />
+      <input
+        type="checkbox"
+        id={id}
+        className={classNames(checkbox)}
+        checked={checked}
+        aria-checked={checked}
+        onChange={onChange}
+        disabled={disabled}
+        aria-labelledby={`${id}-label`}
+      />
+      <div className={label} />
```

Comment:
as discussed in pair session i've quickly fixed the issue from below.

styling changed.
your brach:
![Bildschirmfoto 2025-05-19 um 14 09 18](https://github.com/user-attachments/assets/b1012b26-14a9-4848-b467-c28a46095c95)

develop branch
![Bildschirmfoto 2025-05-19 um 14 09 08](https://github.com/user-attachments/assets/575d2a41-a4db-4593-96bf-b2c44ad01326)